packages/core/src/components/showcase/StoryCard.js

```diff
@@ -0,0 +1,136 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import {
+  Typography,
+  Card,
+  CardActionArea,
+  CardMedia,
+  CardContent,
+  Grid,
+} from "@material-ui/core";
+import { withStyles } from "@material-ui/core/styles";
+
+const styles = (theme) => ({
+  root: {
+    minHeight: `${theme.minHeight}rem`,
```

Comment:
`minHeight` should be a prop of this component... (`theme.minHeight` doesn't tell me this is for which component)